content/doc/upgrade-guide/2.7.adoc

```diff
@@ -0,0 +1,83 @@
+
+=== Upgrading to Jenkins LTS 2.7.4
+
+This release only fixed a regression introduced in 2.7.3. No upgrade notes for this release exist.
+
+=== Upgrading to Jenkins LTS 2.7.3
+
+==== Enforced upgrade of detached plugins
+
+https://issues.jenkins-ci.org/browse/JENKINS-37041[JENKINS-37041]
+
+Jenkins will now ensure that detached plugins, i.e. those whose functionality used to be part of Jenkins core but was moved into a plugin, are always at least the bundled version. A message will be printed to the log informing administrators of this:
+
+ Upgraded detached plugins (and dependencies)
+
+=== Upgrading to Jenkins LTS 2.7.2
+
+==== Support for HTTP_PROXY environment variable in remoting component
+
+https://issues.jenkins-ci.org/browse/JENKINS-32326[JENKINS-32326]
+
+Support for HTTP proxy exclusion in the build agent launcher was added. Build agents that define this variable, but whose connection to the Jenkins master relies on it not being enabled may now fail to to connect to Jenkins. The solution is to fix the proxy configuration as passed to the agent process.
+
+=== Upgrading to Jenkins LTS 2.7.1
+
+==== Setup wizard
+
+The new user experience for Jenkins 2 was overhauled, and instances now start with a setup wizard that sets up a very conservative security configuration and guides users through installation of a recommended set of plugins.
+
+For automatic deployments setting up a preconfigured environment e.g. using https://wiki.jenkins-ci.org/display/JENKINS/Post-initialization+script[+init.groovy+ or +init.groovy.d/+], the System property https://wiki.jenkins-ci.org/display/JENKINS/Features+controlled+by+system+properties[+jenkins.install.runSetupWizard+] allows skipping the setup wizard:
+
+ java -Djenkins.install.runSetupWizard=false -jar jenkins.war
+
+
+==== Servlet 3.1 requirement
+
+Jenkins now requires Servlet API 3.1.
+
+Administrators running Jenkins on non-default servlet containers, such as Tomcat or Websphere, need to ensure the container is capable of Servlet 3.1.
+
+The embedded Winstone-Jetty container has been upgraded to support Servlet 3.1.
+
+
+==== AJP support removed from Winstone-Jetty
+
+The embedded Winstone-Jetty container has been upgraded and dropped support for the AJP protocol. If you're using this protocol for communication between Jenkins and a reverse proxy, you will need to switch to HTTP, or use a different servlet container.
+
+If you installed Jenkins using one of the native packages, it may enable AJP support by default, even if you've never used it. As the related command-line argument to +java -jar jenkins.war+ has been dropped, it is possible that Jenkins will refuse to start after an upgrade, citing an unexpected argument. In that case, you need to edit the service configuration files (e.g. +/etc/default/jenkins+ or +/etc/sysconfig/jenkins+) to remove that parameter.
+
+
+==== Different bundled plugins mechanism
```

Comment:
Maybe makes sense that plugin unbundling changes the offline installation approach ("you cannot get anything excepting the core now. As designed"). Makes sense to reference Juseppe as a workaround